### PR TITLE
all README files on dwyl should now be ampl'd!

### PR DESCRIPTION
**Pull request by des-des**
New ones might not be available because of delay in github api.

http://ampl-demo.herokuapp.com/dwyl/learn-react
http://ampl-demo.herokuapp.com/dwyl/learn-hapi
http://ampl-demo.herokuapp.com/dwyl/adoro
ect ..

### CODE REVIEW

**Comment by des-des** [comment]
some of the functionality could probably be moved to ampl/adoro so this is sort of a first pass..

**Comment by des-des** [comment]
gonna reopen this once I have tests! #4

**Comment by nelsonic** [comment]
@des-des you can keep PRs open, just change the _description_ to something like "**_Don't Merge Yet!**_" so people know not to merge. in fairness the only people watching this are pretty keen on testing and would resist the urge to merge even when the code looks good... :wink:

File: lib/getReadmes.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+var https = require('https');
+var url = require('url');
+var env = require('env2')('./config.env');
```
Comment (nelsonic) [inline lib/getReadmes.js:3]: @des-des if you are using a `.env` file would you mind including a sample in the `README.md` so others who want to contribute to this project know where to start?

File: server/amp.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
+/*!
```
Comment (iteles) [inline server/amp.css:1]: It's worth opening an issue just to log your reasoning in deciding to use Writ :ok_hand:

**Comment by des-des** [comment]
ready to merge @iteles / @nelsonic

**Comment by nelsonic** [comment]
@des-des _amazing_! :sparkles:

**Comment by nelsonic** [comment]
Sadly, travis complained:
screenshot_2016-01-22-19-09-42

**Comment by des-des** [comment]
I think the node version between my computer and yml is different so my arrow functions have crashed it.. 
image

**Comment by nelsonic** [comment]
Yeah, good old ES6...

**Pull request merged**